contrib/firmware_packager/install_dell_bios_exe.py

```diff
@@ -101,8 +112,11 @@ def prompt_reboot():
     ARGS = parse_args()
     CLIENT = Fwupd.Client()
     check_exists(ARGS.exe)
+    is_restore_required = set_conf_only_trusted(CLIENT, False)
     directory = tempfile.mkdtemp()
     guid, deviceid, version = find_uefi_device(CLIENT, ARGS.deviceid)
     cab = generate_cab(ARGS.exe, directory, guid, version)
     install(CLIENT, cab, deviceid, True, True)
+    if is_restore_required:
+        set_conf_only_trusted(CLIENT, True)
```

Comment:
I think it would be good to do a try/except/finally here to ensure that even if the install failed for any reason, that you still restore the configuration back to normal.